### ISSUE TITLE
twisted.web.template.Element documentation gives incorrect usage example for using tags in renderers

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5404]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#5404|
|Type|defect|
|Created|2011-11-22 15:14:55Z|
|Branch|https://github.com/twisted/twisted/tree/twiddle-5404|

The example renderer in the `Element` docstring is:
```
        class Foo(Element):                                                                                               
            @renderer                                                                                                     
            def twiddle(self, request, tag):                                                                              
                return tag['Hello, world.']                    
```

However, this produces:

```
TypeError: 'Tag' object is unsubscriptable
```

<details><summary>Searchable metadata</summary>

```
trac-id__5404 5404
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__branches_twiddle_5404 branches/twiddle-5404
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__web web
keywords__easy_documentation easy documentation
time__1321974895000000 1321974895000000
changetime__1325356579000000 1325356579000000
version__None None
owner__exarkun exarkun
cc__jknight
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [33379]) Branching to 'twiddle-5404'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun commented|
|-|-|

[Build results](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/twiddle-5404)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra set owner to @exarkun|
|-|-|

There's only a single break between the imports and first class in the module, would be nice to fix that, please merge after that.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun commented|
|-|-|

(In [33405]) Add extra whitespace after imports

refs [#5404](https://github.com/twisted/twisted/issues/5404)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_7"></a>|@exarkun set status to `closed`|
|-|-|

(In [33406]) Merge twiddle-5404 

Author: exarkun
Reviewer: thijs
Fixes: [#5404](https://github.com/twisted/twisted/issues/5404)

Correct the `Element` example to use the correct tag API (rather than the old
API provided by Nevow).
